openapi/paths/payout-requests@{id}@payment-instruments.yaml

```
@@ -0,0 +1,62 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Transactions
+  summary: Retrieve payout request's payment instruments
+  operationId: GetPayoutRequestPaymentInstruments
+  x-sdk-operation-name: getPaymentInstruments
+  description: |-
+    Retrieves a list of payment instruments with associated gateway of a payout request with specified ID.
```

nitpick: avoid using block scalars for single line descriptions
```suggestion
  description: Retrieves the payment gateway and a list of payment instruments that are associated with a payout request with a specified ID.
```